app/controllers/idv/image_uploads_controller.rb

```diff
@@ -24,6 +24,7 @@ def image_upload_form
         service_provider: current_sp,
         analytics: analytics,
         uuid_prefix: current_sp&.app_id,
+        irs_attempts_api_tracker: irs_attempts_api_tracker,
```

Comment:
There are currently two implementations for document capture image submission. This is the "synchronous" implementation. The asynchronous implementation isn't yet shipped, and is triggered through [`Api::Verify::DocumentCaptureController`](https://github.com/18F/identity-idp/blob/main/app/controllers/api/verify/document_capture_controller.rb). Will we want to log the event there as well?

Reply:
Not Sure :)

Reply:
I believe we will want to track the asynchronous implementation as well. imo. if we plan to have it turned on for IRS.

Reply:
@mdiarra3 should we imp async event in this PR and Jira ticket? Or should we create a new ticket for this? What would you suggest?

Reply:
I'd do both in the same PR if it's easy

Reply:
not an easy :D, it is enqueuing(DocumentProofingJob) background job and there we need to create irs_attempts_api_tracker instance like (analytics = build_analytics(dcs))

Reply:
ok, seems like a good candidate for a follow-up then

Reply:
Hi @aduth can you help me with how i can test this asynchronous scenario in my local?

Reply:
Circling back to this -- I'd cast another vote for filing a followup story for later. That lets us get this one closed out, and we can come back to the trickier async event separately. The fact that the async version hasn't shipped feels like further reason to separate it out.

Reply:
@olatifflexion You can add the following to `config/application.yml`:

```yaml
doc_auth_enable_presigned_s3_urls: true
```